pkg/config/engine/containerd/config_v1.go

```diff
@@ -163,3 +163,7 @@ func (c *ConfigV1) GetRuntimeConfig(name string) (engine.RuntimeConfig, error) {
 		tree: runtimeData,
 	}, nil
 }
+
+func (c *ConfigV1) EnableCDI() {
+	c.Set("enable_cdi", true)
```

Comment:
I think we should be able to cast `c` to a `Config` type and call `EnableCDI` here instead of reimplementing it.